examples/interaction_energy/sdk/run.py

```diff
@@ -0,0 +1,142 @@
+C 1.4130533000 -1.5536850000 -0.9550667000
+H 2.4258769000 -1.8670780000 -0.7468778000
+C 0.3575976000 -2.0239499000 -0.2530575000
+C 0.4821292000 -3.0179494000 0.8521221000
+H 0.1757705000 -2.5756065000 1.7986281000
+H -0.1601691000 -3.8770412000 0.6639498000
+H 1.5112443000 -3.3572767000 0.9513659000
+C -0.9684711000 -1.5298112000 -0.5939792000
+O -2.0029280000 -1.8396957000 -0.0199453000
+N -0.9956916000 -0.6383870000 -1.6720420000
+H -1.9014057000 -0.2501720000 -1.8985760000
+C 0.0684702000 -0.1191762000 -2.3763759000
+O -0.0397875000 0.7227006000 -3.2531083000
+H 2.0853289000 -0.2760176000 -2.4454577000
+"""
+)
+
+ie_name = "adenine-thymine-stacked-complex-wB97M-V_def2-TZVP"
+job_params = {
+    "name": ie_name,
+    "version": "v1",
+    "kind": "InteractionEnergyCalculation",
+    "parameters": {
+        "molecule_a": {
+            "base64data": moleculeA_adenine,
+            "filetype": "xyz",
+            "params": {
+                "charge": 0,
+                "multiplicity": 1,
+            },
+        },
+        "molecule_b": {
+            "base64data": moleculeB_thymine,
+            "filetype": "xyz",
+            "params": {
+                "charge": 0,
+                "multiplicity": 1,
+            },
+        },
+        "interaction_energy": {
+            "params": {
+                "print_level": 0,
+                "cluster_multiplicity": 1,
+            },
+        },
+        "system": {
+            "params": {
+                "basisname": "def2-tzvp",
+                "methodname": "wb97m-v",
+                "xc_grid_scheme": "SG2",
+            },
+        },
+        "hf": {
+            "params": {
+                "charge": 0,
+                "multiplicity": 1,
+                "g_convergence": 0.000001,
+            },
+            "outputs": {
+                "gradient": False,
+                "polarizability": False,
+                "dipole_derivative": False,
+            },
+        },
+        "est": {
+            "params": {},
+        },
+        "jk_builder": {
+            "type": "core_dfjk",
+            "params": {},
+        },
+        "xc_builder": {
+            "params": {},
+        },
+    },
+    "resources": {
+        "gpu_type": gpu_type,
+    },
+}
+
+prom = PromethiumClient()
+payload = CreateInteractionEnergyCalculationWorkflowRequest(**job_params)
+print(f"Submitting {ie_name}...")
+workflow = prom.workflows.submit(payload)
+
+prom.workflows.wait(workflow.id)
+
+workflow = prom.workflows.get(workflow.id)
+print(f"Workflow {workflow.name} completed with status: {workflow.status}")
+print(f"Workflow completed in {workflow.duration_seconds:.2f}s")
+
+ie_results = prom.workflows.results(workflow.id)
+with open(f"{foldername}/{workflow.name}_results.json", "w") as fp:
+    fp.write(ie_results.model_dump_json(indent=2))
+
+# Numeric results:
+interaction_energy = 627.5095 * ie_results.results["interaction_energy"]["raw_interaction_energy"]
```

Comment:
Be nice to pull this into a constant and have the variable name explain what conversion is being done

Reply:
Done!